### DISCUSSION TITLE
Add "Muted" Metronome as a Gameplay Option / Mod

### DISCUSSION THREAD

**Discussion by NaderYT** [ppy/osu#21915] [Ideas] [answered]
So just playing the Muted mod feels like I'm playing better with a metronome, which is something I always have wanted to try in osu. Now that this mod is here, I want this as an option. Sort of like an added volume bar, or simply an option to disable / enable a metronome without the downside of muting the music when you have a big combo.

**Comment by bdach** [comment, marked as answer]
We've tried metronome only first and it sucked, so it became the muted mod.
